contrib/win32/win32compat/socketio.c

```diff
@@ -118,7 +122,12 @@ socketio_acceptEx(struct w32_io* pio)
 	}
 
 	/* create accepting socket */
+	#ifdef HAVE_AFUNIX_H
+	context->accept_socket = socket(addr.ss_family, SOCK_STREAM, IPPROTO_IP);
+	#else
 	context->accept_socket = socket(addr.ss_family, SOCK_STREAM, IPPROTO_TCP);
+	#endif
```

Comment:
Is this `#ifdef` condition necessary? If `IPPROTO_TCP` does not work for some reason, would it be possible to always use `IPPROTO_IP` (or just `0`)? [man socket(2)](https://man7.org/linux/man-pages/man2/socket.2.html) suggests that specifying `0` (which is the value of `IPPROTO_IP`) will use the default, which again is TCP.

Reply:
I would need to try this out. I don't recall if `IPPROTO_TCP` was not working.